release-notes/5.0/preview/api-diff/preview7/.Net/5.0-preview7_System.Net.md

```diff
@@ -0,0 +1,11 @@
+# System.Net
+
+``` diff
+ namespace System.Net {
+     public class WebHeaderCollection : NameValueCollection, IEnumerable, ISerializable {
+-        public new string this[string name] { get; set; }
```

Comment:
This was removed from ref in PR #37949 ... we still inherit it from [`NameValueCollection`](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Collections.Specialized/src/System/Collections/Specialized/NameValueCollection.cs#L334-L349)

Approved on behalf of @dotnet/ncl